specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/examples/Topology/GetTopologySubscriptionLocation_example.json

```diff
@@ -0,0 +1,68 @@
+{
+    "parameters": {
+        "api-version": "2015-06-01-preview",
+        "subscriptionId": "3eeab341-f466-499c-a8be-85427e154bad",
+        "ascLocation": "centralus"
+    },
+    "responses": {
+        "200": {
+            "body": {
+                "value": [
+                    {
+                        "properties": {
+                            "calculatedDateTime": "2018-07-10T13:56:10.5755270Z",
+                            "resources": [
+                                {
+                                    "resourceId": {
+                                        "ID": "/subscriptions/3eeab341-f466-499c-a8be-85427e154bad/resourceGroups/myservers/providers/Microsoft.Network/virtualNetworks/myvnet"

```

Comment:
field names should be camel case (id)

Reply:
Yes. Example was old version. resourceId is a string, not an object.